Content.Server/Power/Components/ApcPowerReceiverComponent.cs

```diff
@@ -8,11 +9,8 @@ namespace Content.Server.Power.Components
     ///     so that it can receive power from a <see cref="IApcNet"/>.
     /// </summary>
     [RegisterComponent]
-    public sealed partial class ApcPowerReceiverComponent : Component
+    public sealed partial class ApcPowerReceiverComponent : SharedApcPowerReceiverComponent
```

Comment:
Couldnt you just move the whole thing to shared and only network the powered field? pretty sure you would only need to update namespaces, also use AutoNetworkedFIeld

Reply:
Is that fine since most of the other values can only be used on server. They *will* be wrong on client.

Reply:
Only NeedsPower and Recalculate can be moved to shared. Anything else needs server only components to also be moved to shared. So I'm deciding to only put Powered in shared.